app/models/validations/sales/sale_information_validations.rb

```diff
@@ -348,6 +356,18 @@ def validate_mortgage_used_dont_know(record)
     end
   end
 
+  def validate_number_of_staircase_transactions(record)
+    return unless record.numstair
+
+    if record.firststair == 1 && record.numstair >= 2 || record.firststair != 1 && record.numstair == 1
+      record.errors.add :numstair, I18n.t("validations.sales.sale_information.numstair.must_be_greater_than_one")
+      record.errors.add :firststair, I18n.t("validations.sales.sale_information.firststair.more_than_one_transaction")
+    end
+    if record.numstair > 10
```

Comment:
Wouldn't this get handled by max? Would it make sense to keep the error message consistent with whatever the numeric validation message is?

Reply:
Yes it would by default be: `Number of staircasing transactions must be between 2 and 10.`

I can remove this and keep it consistent.